Code/Tools/ProjectManager/Source/ProjectButtonWidget.h

```diff
@@ -115,6 +116,10 @@ namespace O3DE::ProjectManager
         void SetEngine(const EngineInfo& engine);
         void SetProject(const ProjectInfo& project);
         void SetState(enum ProjectButtonState state);
+        const ProjectButtonState& GetState() const
```

Comment:
nitpick: the enum keyword isn't needed on `SetState`

Also this function can return ProjectButtonState by value since it is a 32-bit integral value under the hood.